clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1774,6 +1774,35 @@ def GetMemberOp : CIR_Op<"get_member"> {
   let hasVerifier = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// GetRuntimeMemberOp
+//===----------------------------------------------------------------------===//
+
+def GetRuntimeMemberOp : CIR_Op<"get_runtime_member"> {
+  let summary = "Get the address of a member of a struct";
+  let description = [{
+    The `cir.get_runtime_member` operation gets the address of a member from
+    the input record. The target member is given by a value of type
+    `!cir.data_member` (i.e. a pointer-to-data-member value).
+
```

Comment:
Please mention in the docs how this differs from `GetMemberOp`.

Reply:
Updated.

Reply:
wow, awesome job, thanks!